rezervo/providers/sats/schema.py

```diff
@@ -0,0 +1,72 @@
+from typing import Optional, TypeAlias
+
+from pydantic import BaseModel
+
+SatsLocationIdentifier: TypeAlias = int
+
+
+class SatsClassImage(BaseModel):
+    alt: str
+    src: str
+
+
+class SatsClassDetail(BaseModel):
+    clubName: str
+    duration: int
+    durationText: str
```

Comment:
This should probably be used to calculate `end_time` if they ever use anything other than minutes

Reply:
The trouble here is that we have no idea what this could result to otherwise. It could be "hr", "hrs", "hour", "hours", "time" or "timer", since it seems like a presentational text. (see the image below)

Since I cannot find any classes with durationText != "min" on all the locations I test with and there are classes up to 75 minutes _not_ using "hours", I think it is safe to ignore this.
![Screenshot 2024-02-20 at 14 52 42](https://github.com/mathiazom/rezervo/assets/26925695/ba64cb94-a665-4c9b-bfeb-177288fcf448)